pkgs/development/python-modules/goocalendar/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, python2Packages, pkgconfig, gobjectIntrospection, gtk3, goocanvas2 }:
```

Comment:
For python packages, please don't pass `pythonPackages` as argument, but `fetchPypi`, `pygtk`, ...

If this is Python 2 only, you need to add `disabled = isPy3k`.